src/index/index.h

```diff
@@ -62,6 +77,13 @@ void Index_AddField
 	const char *field  // field to add
 );
 
+// adds field to index
+void Index_AddFullTextField
+(
+	Index *idx,
+	IndexField *field  // field to add
+);
+
```

Comment:
Wouldn't you prefer to have a single `AddField` function which only accepts `IndexField`
this function can serve both `ExactMatch` and `FullText` indexes

Reply:
no because I want to be able to add simple field without initialize all fields options